app/demo-loop/Runner.hh

```diff
@@ -101,7 +101,6 @@ class Runner
 
   private:
     //// DATA ////
-
```

Comment:
I kind of prefer blank lines after these grouping separators: is this whitespace change consistent with what's in the rest of the code?

Reply:
I don't think you need to refactor; this just looks like a standalone change (probalby introduced by adding something and deleting it) that is unnecessary.

Reply:
Yes. My point is that there are classes (ProcessBuilder, GeantModelImporter, and others) with no blank lines. So I could make all of them consistent now.